.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+### 🗒 Description
+
+If helpful for the review process, please describe your changes from a technical or non technical perspective
+
+#
+### 🎫 Git ticket reference
+
+- 
+
+#
+### 📷 Demo screen recording or Screenshots
```

Comment:
Most PRs don't require this (unless its soully UI changes), I wouldn't include it in the template and make it completely optional.